plugins/modules/zos_mvs_raw.py

```diff
@@ -1900,13 +1907,23 @@ def run_module():
             )
             response = build_response(program_response.rc, dd_statements, program_response.stdout)
             result = combine_dicts(result, response)
-            if program_response.rc != 0 :
+
+            if program_response.rc != 0 and max_rc is None:
```

Comment:
@fernandofloresg - Am I interpreting your comment correctly in that you are asking what the response contains such that the test case should inspect more than a return code. 

For example, some of our tests cases have succeeded only because there were not enough detailed assertions happening and by adding more assertions we can protect the code from future changes.